css/index.css

```diff
@@ -1,124 +1,221 @@
-/*   NAVBAR STYLING  */
-#navbar {
-  display: block;
-  letter-spacing: 1.5px;
-  word-spacing: 5px;
-  margin-inline: 30px;
-  margin-block: 2em;
-  text-align: right;
-}
+@import url("https://fonts.googleapis.com/css2?family=Belanosima:wght@600&family=Roboto+Mono:wght@300;400&display=swap");
 
-/*  NAV LINK STYLING  */
-a {
-  text-decoration: none;
-  font-weight: 600;
+:root {
+  --primary-color: #13435b;
+  --secondary-color: #052230;
+  --off-white: #fdfbeb;
+  --teal: #298c6f;
+  --lime: #bedb45;
+  --orange: #f97821;
 }
-a:hover, a:active {
-  color: #bedb45;
+html {
+  scroll-behavior: smooth;
+  background-color: var(--primary-color);
+  box-sizing: border-box;
 }
 
+*,
+::before,
+::after {
+  padding: 0;
+  margin: 0%;
+}
 
-/*     MAIN    */
 body {
-  background-color: #13435b;
-  font-family: 'Roboto Mono', monospace;
-  padding: 0px;
-  margin: 0%;
+  font-family: "Roboto Mono", monospace;
 }
 
-h1, h2, h4 {
-  font-family: 'Belanosima', sans-serif;
+h1, h2, h3 {
+  font-family: "Belanosima", sans-serif;
   font-weight: 600;
 }
 
-h2, h4, a {
-  color: #298c6f;
+h2, h3, a {
+  color: var(--teal);
+}
+
+p {
+  color: var(--off-white);
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 1.5;
+  margin: 2% 15%;
+  width: 70%;
+}
+@media only screen and (max-width: 768px) {
+  p {
+    font-size: .8rem;
+    margin: 5% 10%;
+    width: 80%;
+    font-size: 1rem;
+  }
 }
 
 h1 {
-  color: #bedb45;
-  font-size: 72px;
-  margin-inline: 15%;
+  color: var(--lime);
+  font-size: 5.5rem;
+  margin: 2% 0 0 15%;
+  line-height: 6rem;
+  letter-spacing: 1.5px;
+  width: 50%;
+}
+@media only screen and (max-width: 768px) {
```

Comment:
It's probably better to have just one media section, so there's no risk of widths getting out of sync. You'll see "DRY" as a general principle: "Don't Repeat Yourself".